module3/homework/advanced_calculator/advancedCalculator.cpp

```diff
@@ -0,0 +1,177 @@
+#include "advancedCalculator.hpp"
+
+#include <algorithm>
+#include <cmath>
+#include <iostream>
+#include <regex>
+
+auto addAction = [](double firstNumber, double secondNumber) {
+    return firstNumber + secondNumber;
+};
+
+auto substractAction = [](double firstNumber, double secondNumber) {
+    return firstNumber - secondNumber;
+};
+
+auto divideAction = [](double firstNumber, double secondNumber) {
+    return firstNumber / secondNumber;
+};
+
+auto multiplyAction = [](double firstNumber, double secondNumber) {
+    return firstNumber * secondNumber;
+};
```

Comment:
not needed

Reply:
just use a + b etc..

Reply:
i cant delete this lambdas becasue all actions methods are in special map(action, method), then i call this methods, and also i dont know how do it in another way :)
```suggestion
specMap::iterator it = actionsHolder.find(*action);
    auto result = it->second(firstNumber, secondNumber);
```